### PR TITLE
A fix for the issue: Unable to publish resume #189

### PR DESCRIPTION
**Pull request by utkarsh-shekhar**
I wrapped the async code inside async.series which fixes the issue.

### CODE REVIEW

**Comment by utkarsh-shekhar** [comment]
Not sure why the test failed. That particular one seems to pass for me.
I'm new to this so if it is my mistake please do let me know.
screenshot from 2017-07-06 02-36-02

File: lib/publish/index.js [hunk 1]
```diff
@@ -90,7 +98,7 @@ module.exports = function publish(resumeJson, program, config) {
                             default: defaultEmail
                         }, function(err, email) {
 
-                            next(err, email)
+                            next(err, email);
```
Comment (PeterDaveHello) [inline lib/publish/index.js]: This is not related.

File: lib/publish/index.js [hunk 2]
```diff
@@ -48,34 +48,42 @@ module.exports = function publish(resumeJson, program, config) {
                 break;
 
             case customPublish:
-
```
Comment (PeterDaveHello) [inline lib/publish/index.js]: Let's keep this line to minimize the diff.
